app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsViewModel.kt

```diff
@@ -17,14 +17,24 @@ class MovieDetailsViewModel(
     private val movieDetailsMapper: MovieDetailsMapper,
     private val movieId: Int
 ) : ViewModel() {
-    val movieDetailsViewState: StateFlow<MovieDetailsViewState> =
-        movieRepository.movieDetails(movieId)
-            .map { movies -> movieDetailsMapper.toMovieDetailsViewState(movies) }
-            .stateIn(
-                viewModelScope, SharingStarted.Eagerly, movieDetailsMapper.toMovieDetailsViewState(
-                    MoviesMock.getMovieDetails(movieId)
-                )
-            )
+    private val initialViewState = MovieDetailsViewState(
+        id = 1,
+        imageUrl = null,
+        voteAverage = 8.1f,
+        title = "",
+        overview = "",
+        isFavorite = false,
+        crew = emptyList(),
+        cast = emptyList()
+    )
```

Comment:
You can define `EMPTY` in `MovieDetailsViewState` companion object and instead of creating `initialViewState` here, just call `MovieDetailsViewState.EMPTY`